app/controllers/Subscriptions.scala

```diff
@@ -139,7 +142,17 @@ class Subscriptions(
     val css = "paperSubscriptionLandingPage.css"
     val canonicalLink = Some(buildCanonicalPaperSubscriptionLink())
 
-    Ok(views.html.main(title, id, js, css, None, canonicalLink)).withSettingsSurrogateKey
+    def pricesToMap(prices: List[ZuoraCatalogPricePlan], prefix: String) = {
+      prices.map(price => (
```

Comment:
Small syntactical thing with Scala:

When using map/filter etc on a single line you would write it as so with brackets:

`foo.map(f => ...)`

Whereas when over multiple lines you would use curly braces like so:

```
foo.map { f => 
...
}
```